scripts/serverSetup.md

```diff
@@ -0,0 +1,66 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 and Maven 3+ are installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME**
+
+## Deploying the web sever
+In the yw_website directory, change directory into the scripts directory. You should see a number of bash scripts:
+```
+deploy_python.sh
+deploy_python3.sh
+remote_deploy_python.sh
+remote_deploy_python3.sh
+workflow_sample.gv
```

Comment:
This shouldn't have been in the scripts folder. TODO: Remove this from folder and write up.
Also the remote deploy scripts need to be removed.